docs/pages/choose-an-edition/teleport-team.mdx

```diff
@@ -39,194 +41,250 @@ web & tsh--Client traffic-->proxy
 <Notice type="tip">
 
 For a detailed comparison of Teleport's editions, including how Teleport Team
-compares to Teleport Team, see the [Choose an Edition](introduction.mdx) guide.
+compares to Teleport Enterprise Cloud and Teleport Enterprise, see 
+[Choose an Edition](introduction.mdx).
 
 </Notice>
 
-## Get started with Teleport Team
+This guide explains how to register a local server with a
+Teleport Team account. After you register the server, you can access it through the 
+Web UI in a browser or using the terminal. You can also record your sessions, so you 
+can review it later.
 
-This guide walks you through the process of registering a local server with a
-Teleport Team account. You can then access the server through the browser and
-the terminal, and record your session so you can review it later.
-
-### Prerequisites
+## Prerequisites
 
 - A Teleport Team account. You can sign up for a free trial at the following
   page:
 
   https://goteleport.com/signup/
 
-- Docker installed on your workstation. We will use a Docker container to
-  register a server with Teleport via the Teleport SSH Service. You can find
-  installation instructions on [Docker's
-  website](https://docs.docker.com/get-docker/).
+  After you sign up, you receive an email invitation to activate your account.
+  Use this account to authenticate your identity when you access the Teleport
+  Team tenant.
 
-  <Notice type="tip">
+- Docker installed on your workstation. 
 
-  Docker is only required for the local demo environment we use in this guide.
-  If you want to get started registering your servers in your infrastructure
-  with Teleport, view our documentation on how to [register servers and
-  configure access to them](../server-access/introduction.mdx).
+  This guide illustrates how to register a server with Teleport Team using a Docker 
+  and the Teleport SSH Service. Docker is only required for the local demo environment 
+  used in this guide. You can find installation instructions for Docker on [Docker's
+  website](https://docs.docker.com/get-docker/). If you want to register servers in 
+  Teleport Team without using Docker, see 
+  [register servers and configure access to them](../server-access/introduction.mdx).
```

Comment:
```suggestion
  [our guide to enrolling servers with Teleport](../server-access/introduction.mdx).
```

Reply:
I don't like the misleading inline text link. When I follow this link, I don't find "our guide to enrolling servers with Teleport" or even "Enrolling servers in Teleport". Best practice is to link to actual titles/headings, but some of the titles/headings aren't that useful, so I'm punting here for now.